airbyte-webapp/src/views/Connection/ConnectionForm/calculateInitialCatalog.ts

```diff
@@ -119,13 +119,13 @@ const getOptimalSyncMode = (
 const calculateInitialCatalog = (
   schema: SyncSchema,
   supportedDestinationSyncModes: DestinationSyncMode[],
-  isEditMode?: boolean
+  isNotCreateMode?: boolean
```

Comment:
This is slightly more clear as isEditMode was true if mode was readonly. That was correct behavior, but now it's more explicit.
Ideally we should invert this whole boolean to `isCreateMode` but that expanded the work a little beyond what I'd like.